oxidation/libparsec/crates/testbed/src/env.rs

```diff
@@ -0,0 +1,171 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
+
+use super::*;
+
+// Testbed always works in memory, the path only acts as an identifier.
+const TESTBED_BASE_DUMMY_PATH: &str = "/parsec/testbed/";
+
+#[derive(Debug, PartialEq)]
+pub enum TestbedKind {
+    ClientServer,
+    ClientOnly, // Server is considered offline for the whole run
+}
+
+#[derive(Debug)]
+pub struct TestbedEnv {
+    pub kind: TestbedKind,
+    // Config dir is used as the key identifying the testbed env
+    // note testbed is designed to always run mocked in memory
```

Comment:
```suggestion
    /// Config dir is used as the key identifying the testbed env
    /// note testbed is designed to always run mocked in memory
```

Convert them to doc comment to provide that information to the dev